cdragontoolbox/tftdata.py

```diff
@@ -65,17 +67,70 @@ def replace_in_data(data):
                     if key in data and data[key] in replacements:
                         data[key] = replacements[data[key]]
 
-            for data in template["champs"]:
+            for data in [template["sets"][x]["traits"] for x in template["sets"]]:
+                replace_in_data(data)
+            for data in [template["sets"][x]["champions"] for x in template["sets"]]:
```

Comment:
- No need to create a list, the generator is sufficient
- You can iterate on values directly
```py
for data in (x["champions"] for x in template["sets"].values()):
```
(Same for the two `for` loops)